addons/website_event_track_live/static/src/js/website_event_track_live.js

```diff
@@ -29,30 +26,27 @@ publicWidget.registry.websiteEventTrackLive = publicWidget.Widget.extend({
     //--------------------------------------------------------------------------
 
     _onPlayerReady: function () {
-        $(window).on('resize', this._onResize.bind(this));
```

Comment:
Why did you remove that?
Doesn't really say in your commit message.

Reply:
Actually, we don't need to add a listener here. When the video ends, the event `video-ended` will be triggered and the function `_onVideoEnded` will be called. The function will instantly add a cover over the Youtube player (in non-fullscreen mode). The function will then fetch recommendations from the server and add them to the cover.